shared/src/main/scala/com/github/takayahilton/sqlformatter/UnsafeSqlFormatter.scala

```diff
@@ -0,0 +1,52 @@
+package com.github.takayahilton.sqlformatter
+
+import com.github.takayahilton.sqlformatter.core.FormatConfig
+import com.github.takayahilton.sqlformatter.languages._
+
+object UnsafeSqlFormatter {
```

Comment:
Is this needed?
If you only need it for testing, I would like to remove this and fix the existing test.

Reply:
Yes it was for testing purpose, but this can be done differently for sure.